contest/contributions/john-coene.yml

```diff
@@ -0,0 +1,19 @@
+# Contribution's title, author, abstract
+title: A Coronavirus Tracker, API & Crawler
+author: John Coene
+abstract: |
+  A tracker that uses three different sources of data to present the user with
+  a mobile-first application; the app, the API and its data crawler are both open-source.
+
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: Coronavirus Tracker
+# URL of the public source code repository
+repository: https://github.com/JohnCoene/coronavirus
+# Type of contribution and how it can be featured as gallery content
+# Keep only the type and content relevant to your contribution
+# For a Shiny app, provide its URL:
+type: shiny
+content:  https://shiny.john-coene.com/coronavirus/
+# For a general online-published resource (e.g. a website / report):
+type: webpage
+content: https://coronavirus.john-coene.com/
```

Comment:
@JohnCoene, Thanks for your submission.

Note that only one of the two `content:` should be kept, either the main landing `webpage` or directly the `shiny` app.
The URL will define what is embedded and displayed in the CovidR gallery website, see e.g. the example at https://milano-r.github.io/erum2020-covidr-contest/author-contribution-example.html.

It is probably more appealing to have the Shiny app embedded in there, also given it contains the relevant links in the sidebar (in this regard, the API link http://shiny.john-coene.com:8080/__swagger__/ seems broken).

You can either push an updated `.yml` file with your choice or reply to this thread and we'll take care of it.